CHANGELOG.md

```diff
@@ -56,6 +56,10 @@ Version <dev> adds Apache Kafka instrumentation for the rdkafka and ruby-kafka g
 
 [PR#2851](https://github.com/newrelic/newrelic-ruby-agent/pull/2851)
 
+- **Feature: Collect just MIME type for AcionDispatch 7.0+ requests**
```

Comment:
Should we call this out in the overarching list at the top of the log?

Reply:
OH yes 🤦🏽‍♀️

Reply:
done https://github.com/newrelic/newrelic-ruby-agent/pull/2855/commits/7beeceabe75e13b45e82c6196d8c88645a337e07